backends/dpdk/dpdk.def

```diff
@@ -304,27 +304,6 @@ class DpdkMovStatement : DpdkUnaryStatement {
 #noconstructor
 }
 
-class DpdkNegStatement: DpdkUnaryStatement {
```

Comment:
how come all these are no longer necessary?

Reply:
These were added in the initial commit for p4c-dpdk. I presume these were just placeholder instructions since the development of p4-dpdk  software pipeline and compiler was going in parallel. The functionality of all these (now removed) operations are achieved using other existing instructions in the p4-dpdk pipeline.